engine/src/test/java/org/hibernate/search/test/util/impl/FileHelperTest.java

```diff
@@ -48,19 +50,19 @@ public static void prepareRootDirectory() {
 		if ( buildDir == null ) {
 			buildDir = ".";
 		}
-		root = new File( buildDir, "filehelper" );
-		log.infof( "Using %s as test directory.", root.getAbsolutePath() );
+		root = Paths.get( buildDir, "filehelper" );
+		log.infof( "Using %s as test directory.", root.toAbsolutePath() );
 	}
 
-	private File createFile(File dir, String name) throws IOException {
-		File file = new File( dir, name );
-		file.createNewFile();
+	private Path createFile(Path dir, String name) throws IOException {
+		Path file = dir.resolve( name );
+		Files.createFile( file );
 		writeDummyDataToFile( file );
 		return file;
 	}
 
-	private void writeDummyDataToFile(File file) throws IOException {
-		FileOutputStream os = new FileOutputStream( file, true );
+	private void writeDummyDataToFile(Path file) throws IOException {
+		FileOutputStream os = new FileOutputStream( file.toFile(), true );
```

Comment:
I think the nio way would be: `Files.newOutputStream( file, StandardOpenOption.APPEND )`;